src/corehost/build.sh

```diff
@@ -22,13 +22,14 @@ init_rid_plat()
         __rid_plat=""
         if [ -e /etc/os-release ]; then
             source /etc/os-release
-            if [[ "$ID" == "rhel" && $VERSION_ID = 7* ]]; then
-                __rid_plat="rhel.7"
-            elif [[ "$ID" == "centos" && "$VERSION_ID" = "7" ]]; then
-                __rid_plat="rhel.7"
-            else
-                __rid_plat="$ID.$VERSION_ID"
+            if [[ "$ID" == "centos" ]]; then
+                ID="rhel"
+            fi
+            if [[ "$ID" == "rhel" || "$ID" == "alpine" ]]; then
+                # remove the minor version digit 
```

Comment:
For alpine, it isn't the "minor" version digit.  So maybe this comment should say "remove the last version digit"